### DISCUSSION TITLE
About time stamp in OVP800

### DISCUSSION THREAD

**Discussion by Astronomia-maker** [ifm/ifm3d#393] [Q&A]
Hello,

I tried to set current date and time in OVP800 but as i dont have any root access i cant. So how can i set current date and time on OVP800. Because OVP800 has 2020 year time on host machine while connect via SSH. I need device current time for my logs.

**Comment by lola-masson** [comment]
Hi @Astronomia-maker,
You can use: 
```bash
o3r-vpu-c0:~$ sudo date --set "Tue Apr 18 2023 13:21"
```
Alternatively, you can setup a SNTP server/client in the json configuration, see [sntps setup](https://ifm3d.com/documentation/DeviceConfiguration/sntp.html).

**Reply by Astronomia-maker** [reply]
Hi @lola-masson, 

Thank you for the answer. I executed suggested command and it works but when i restart the device, the device timestamp is gone and set back to default to UTC 2020. And for SNTP server, i dont have any internet connection available for my application

**Reply by graugans** [reply]
That's due to the fact we have no battery backed RTC on the OVP8xx Like @lola-masson stated the sntp maybe a solution.  You actually don't need an internet connection. The only thing you need is an PC or similar device with a battery backed RTC. This document may help https://github.com/ifm/ifm3d/blob/main/doc/ntp-setup.md it is related to the O3D3xx but the PC part shall also work with the O3R.

**Reply by graugans** [reply]
Other users of our cameras had success with using this for the time translation: https://github.com/ethz-asl/cuckoo_time_translator
